### ISSUE TITLE
bash: make: command not found

### ISSUE THREAD

**Issue by smoms** [docker/dev-environments#60]
followed the example:

https://docs.docker.com/desktop/dev-environments/#start-a-single-container-dev-environment

but I get **bash: make: command not found** error.

**Comment by glours** [comment]
Hello @smoms 
I just change the base image use by the `single-dev-environment` sample, can you try to recreate a new dev env from the sample and check everything is ok now, please?

**Comment by smoms** [comment]
@glours now the image looks ok. Once the sample app starts though, once I open the browser on http://localhost:8080/ it hangs for ever...

**Comment by glours** [comment]
Yes, it seems there is some issues on the VsCode side with port forwarding since the `1.60.0` version
https://github.com/microsoft/vscode-remote-release/issues/5551

Can you check that you can curl the server directly from a terminal in VsCode with `curl http://localhost:8080`?
